docs/quickstart.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 sidebar_position: 1
-title: Quick Start
+title: Getting Started
```

Comment:
We refer to it as "quick start" in various other pages and external sites (e.g., Github READMEs). I would keep the quick start wording, but if not then pay attention to make the wordings consistent both in the docs and READMEs.